src/test/java/marquez/client/MarquezClientTest.java

```diff
@@ -240,10 +245,12 @@ public void testCreateDataset() throws Exception {
     final URL url = buildUrlFor(path);
     when(http.url(pathTemplate, NAMESPACE_NAME)).thenReturn(url);
 
-    final DatasetMeta meta =
-        DatasetMeta.builder()
+    final DbTableMeta meta =
```

Comment:
Wasn't sure if we wanted to test each concrete class of DatasetMeta here.
Your thoughts @wslulciuc ?

Reply:
Great question. Yes, we'll want to have tests for each dataset type. I would slit **testCreateDataset()** into two tests: **testCreateDbTableDataset()** and **testCreateStreamDataset()**. This will ensure the client properly serializes each type correctly.

Reply:
I kinda thought that was what you would want :)

Reply:
This test has been split into two now.

Reply:
Thanks!